Test/WebTestCase.php

```diff
@@ -346,9 +353,51 @@ protected function postFixtureSetup()
     }
 
     /**
-     * Callback function to be executed after Schema restore.
+     * Callback function to be executed after Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function postFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before Schema restore
+     *
+     * @param ObjectManager $manager The object manager
+     * @param ProxyReferenceRepository $referenceRepository The reference repository
+     * @return WebTestCase
+     */
+    protected function preFixtureRestore(ObjectManager $manager, ProxyReferenceRepository $referenceRepository)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed after save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
+     */
+    protected function postReferenceSave(ObjectManager $manager, AbstractExecutor $executor, $backupFilePath)
+    {
+
+    }
+
+    /**
+     * Callback function to be executed before save of references
+     *
+     * @param ObjectManager $manager The object manager
+     * @param AbstractExecutor $executor Executor of the data fixtures
+     * @param string $backupFilePath Path of file used to backup the references of the data fixtures
+     * @return WebTestCase
      */
-    protected function postFixtureRestore()
```

Comment:
OKey, I will remove those arguments, but... it will be good to access instance of the ```ObjectManager``` or ```ProxyReferenceRepository``` directly in ```postFixtureRestore()``` method. What do you think about it?